### PR TITLE
Add the OCP-15603 ocp-15605

### PR DESCRIPTION
**Pull request by emmajiafan**
Add the auto scrip OCP-15603 ocp-15605 to verification-tests.
runner v3 test log:
 # @case_id OCP-15605
  @admin @destructive
  Scenario: Create/get/update/delete for ServiceBinding resource                          # features/svc-catalog_asb/svc-catalog.feature:280
      [05:06:33] INFO> === Before Scenario: Create/get/update/delete for ServiceBinding resource ===
      [05:06:33] INFO> === End Before Scenario: Create/get/update/delete for ServiceBinding resource ===
...
      [05:07:53] INFO> HTTP DELETE took 0.402 sec: 200 OK | application/json 235 bytes
      
      [05:07:53] INFO> Shell Commands: rm -r -f -- /home/jenkins/workspace/Runner-v3/workdir
      
      [05:07:53] INFO> Exit Status: 0
      [05:07:53] INFO> === End After Scenario: Create/get/update/delete for ServiceBinding resource ===

waiting for operation up to 3600 seconds..
waiting for operation up to 3600 seconds..
waiting for operation up to 3600 seconds..
waiting for operation up to 3600 seconds..
waiting for operation up to 3600 seconds..
waiting for operation up to 3600 seconds..
waiting for operation up to 3600 seconds..
  
  # @case_id OCP-15603
  @admin @destructive
  Scenario: Create/get/update/delete for ServiceInstance resource                   # features/svc-catalog_asb/svc-catalog.feature:214
      [05:07:56] INFO> === Before Scenario: Create/get/update/delete for ServiceInstance resource ===
      [05:07:56] INFO> Shell Commands: mkdir -v -p '/home/jenkins/workspace/Runner-v3/workdir'
      mkdir: created directory '/home/jenkins/workspace/Runner-v3/workdir'
      ...
      [05:08:59] INFO> Exit Status: 0
      [05:08:59] INFO> waiting up to 30 seconds for user clean-up to take place
      [05:09:06] INFO> REST delete_oauthaccesstoken for user 'BushSlicer::APIAccessor:testuser-38@ocp4', base_opts: {:options=>{:api_version=>"v1", :accept=>"application/json", :content_type=>"application/json", :oauth_token=>"uJDBTj1IHrPuBxZjkWyCb-_588re4CYF48XuIBJZqw8"}, :base_url=>"https://api.yinzhou.qe.devcluster.openshift.com:6443", :headers=>{"Accept"=>"<accept>", "Content-Type"=>"<content_type>", "Authorization"=>"Bearer <oauth_token>"}}, opts: {:token_to_delete=>"uJDBTj1IHrPuBxZjkWyCb-_588re4CYF48XuIBJZqw8"}
      [05:09:06] INFO> HTTP DELETE https://api.yinzhou.qe.devcluster.openshift.com:6443/apis/oauth.openshift.io/v1/oauthaccesstokens/uJDBTj1IHrPuBxZjkWyCb-_588re4CYF48XuIBJZqw8
      [05:09:07] INFO> HTTP DELETE took 0.422 sec: 200 OK | application/json 235 bytes
      
      [05:09:07] INFO> Shell Commands: rm -r -f -- /home/jenkins/workspace/Runner-v3/workdir
      
      [05:09:07] INFO> Exit Status: 0
      [05:09:07] INFO> === End After Scenario: Create/get/update/delete for ServiceInstance resource ===

2 scenarios (2 passed)
49 steps (49 passed)
2m33.634s

### CODE REVIEW

**Comment by chengzhang1016** [comment]
LGTM. Thanks.

File: features/svc-catalog_asb/svc-catalog.feature [hunk 1]
```diff
@@ -207,6 +207,152 @@ Feature: Service-catalog related scenarios
...
+  Scenario: Create/get/update/delete for ServiceInstance resource
+    Given I have a project
+    And evaluation of `project.name` is stored in the :ups_broker_project clipboard
+    And I create a new project
+    And evaluation of `project.name` is stored in the :user_project clipboard
+
+    # Deploy ups broker
+    Given I register clean-up steps:
+    """
+    I run the :delete admin command with:
```
Comment (pruan-rht) [inline features/svc-catalog_asb/svc-catalog.feature]: @emmajiafan I think we can replace this with `admin ensure...`

File: features/svc-catalog_asb/svc-catalog.feature [hunk 2]
```diff
@@ -207,6 +207,152 @@ Feature: Service-catalog related scenarios
...
+  Scenario: Create/get/update/delete for ServiceBinding resource
+    Given I have a project
+    And evaluation of `project.name` is stored in the :ups_broker_project clipboard
+    And I create a new project
+    And evaluation of `project.name` is stored in the :user_project clipboard
+
+    # Deploy ups broker
+    Given I register clean-up steps:
+    """
+    I run the :delete admin command with:
```
Comment (pruan-rht) [inline features/svc-catalog_asb/svc-catalog.feature]: you can use `admin ensure` step here

File: features/svc-catalog_asb/svc-catalog.feature [hunk 3]
```diff
@@ -207,6 +207,152 @@ Feature: Service-catalog related scenarios
...
+      | resource   | servicebinding/ups-binding |
+      | show_label | true                       |
+    Then the output should contain "app=test-binding"
+
+    # Delete servicebinding
+    When I run the :delete client command with:
+      | object_type       | servicebinding |
+      | object_name_or_id | ups-binding    |
+    Then the step should succeed
+    Given I ensure "ups-binding" servicebinding is deleted
```
Comment (pruan-rht) [inline features/svc-catalog_asb/svc-catalog.feature]: the `I ensure` step can replace the line above (349-352)

File: features/svc-catalog_asb/svc-catalog.feature (hunk 1 above)
Comment (emmajiafan) [reply features/svc-catalog_asb/svc-catalog.feature]: Updated

File: features/svc-catalog_asb/svc-catalog.feature (hunk 2 above)
Comment (emmajiafan) [reply features/svc-catalog_asb/svc-catalog.feature]: Updated

File: features/svc-catalog_asb/svc-catalog.feature (hunk 3 above)
Comment (emmajiafan) [reply features/svc-catalog_asb/svc-catalog.feature]: Updated

**Comment by emmajiafan** [comment]
> with line 274, you can remove 270-273

**Comment by pruan-rht** [comment]
@emmajiafan LG, do you mind attaching a run log for the latest modification?

**Comment by emmajiafan** [comment]
@pruan-rht , the run log for the latest modification:
 # @case_id OCP-15603
  @admin @destructive
  Scenario: Create/get/update/delete for ServiceInstance resource                   # features/svc-catalog_asb/svc-catalog.feature:214
      [09:14:38] INFO> === Before Scenario: Create/get/update/delete for ServiceInstance resource ===
      [09:14:38] INFO> === End Before Scenario: Create/get/update/delete for ServiceInstance resource ===
    Given I have a project                                                          # features/step_definitions/project.rb:7
      [09:14:38] INFO> HTTP GET https://api.wsun-44ci.qe.gcp.devcluster.openshift.com:6443/.well-known/oauth-authorization-server
      [09:15:35] INFO> === After Scenario: Create/get/update/delete for ServiceInstance resource ===
      [09:15:35] INFO> cleaning-up user testuser-48 projects
      [09:15:36] INFO> Shell Commands: oc delete projects --all --config=/home/jenkins/workspace/Runner-v3/workdir/ocp4_testuser-48.kubeconfig
      project.project.openshift.io "44ywv" deleted
      project.project.openshift.io "e8rmm" deleted
      
      [09:15:37] INFO> Exit Status: 0
      [09:15:37] INFO> waiting up to 30 seconds for user clean-up to take place
      [09:15:43] INFO> REST delete_oauthaccesstoken for user 'BushSlicer::APIAccessor:testuser-48@ocp4', base_opts: {:options=>{:api_version=>"v1", :accept=>"application/json", :content_type=>"application/json", :oauth_token=>"TSNsp_bBzQ9iZl93xMwCOR2lQVT9i4gkGky15FzITK0"}, :base_url=>"https://api.wsun-44ci.qe.gcp.devcluster.openshift.com:6443", :headers=>{"Accept"=>"<accept>", "Content-Type"=>"<content_type>", "Authorization"=>"Bearer <oauth_token>"}}, opts: {:token_to_delete=>"TSNsp_bBzQ9iZl93xMwCOR2lQVT9i4gkGky15FzITK0"}
      [09:15:43] INFO> HTTP DELETE https://api.wsun-44ci.qe.gcp.devcluster.openshift.com:6443/apis/oauth.openshift.io/v1/oauthaccesstokens/TSNsp_bBzQ9iZl93xMwCOR2lQVT9i4gkGky15FzITK0
      [09:15:44] INFO> HTTP DELETE took 0.302 sec: 200 OK | application/json 235 bytes
      
      [09:15:44] INFO> Shell Commands: rm -r -f -- /home/jenkins/workspace/Runner-v3/workdir
      
      [09:15:44] INFO> Exit Status: 0
      [09:15:44] INFO> === End After Scenario: Create/get/update/delete for ServiceInstance resource ===


  # @case_id OCP-15605
  @admin @destructive
  Scenario: Create/get/update/delete for ServiceBinding resource                          # features/svc-catalog_asb/svc-catalog.feature:269
      [09:15:47] INFO> === Before Scenario: Create/get/update/delete for ServiceBinding resource ===
      [09:15:47] INFO> Shell Commands: mkdir -v -p '/home/jenkins/workspace/Runner-v3/workdir'
      
      [09:16:42] INFO> Exit Status: 0
      [09:16:42] INFO> waiting up to 30 seconds for user clean-up to take place
      [09:16:50] INFO> REST delete_oauthaccesstoken for user 'BushSlicer::APIAccessor:testuser-48@ocp4', base_opts: {:options=>{:api_version=>"v1", :accept=>"application/json", :content_type=>"application/json", :oauth_token=>"_6vB_oaMrK-kQN5BP5wbMdqblgP08TkVuAQ4jHn_r74"}, :base_url=>"https://api.wsun-44ci.qe.gcp.devcluster.openshift.com:6443", :headers=>{"Accept"=>"<accept>", "Content-Type"=>"<content_type>", "Authorization"=>"Bearer <oauth_token>"}}, opts: {:token_to_delete=>"_6vB_oaMrK-kQN5BP5wbMdqblgP08TkVuAQ4jHn_r74"}
      [09:16:50] INFO> HTTP DELETE https://api.wsun-44ci.qe.gcp.devcluster.openshift.com:6443/apis/oauth.openshift.io/v1/oauthaccesstokens/_6vB_oaMrK-kQN5BP5wbMdqblgP08TkVuAQ4jHn_r74
      [09:16:50] INFO> HTTP DELETE took 0.304 sec: 200 OK | application/json 235 bytes
      
      [09:16:50] INFO> Shell Commands: rm -r -f -- /home/jenkins/workspace/Runner-v3/workdir
      
      [09:16:50] INFO> Exit Status: 0
      [09:16:50] INFO> === End After Scenario: Create/get/update/delete for ServiceBinding resource ===

2 scenarios (2 passed)
45 steps (45 passed)

**Pull request merged**